docker/lnd/start-lnd.sh

```diff
@@ -46,7 +46,6 @@ NETWORK=$(set_default "$NETWORK" "simnet")
 CHAIN=$(set_default "$CHAIN" "bitcoin")
 
 lnd \
-    --datadir="/data" \
```

Comment:
I'd also argue that the `logdir` itself doesn't need to be modified either.

Reply:
Yeah, probably remove the logdir flag also